mono/mini/Makefile.am.in

```diff
@@ -725,9 +728,9 @@ FULLAOT_LIBS = \
 fullaotcheck: $(mono) $(fullaot_regtests)
 	rm -rf fullaot-tmp
 	mkdir fullaot-tmp
-	$(MAKE) fullaot-libs AOT_FLAGS=full GSHAREDVT=$(GSHAREDVT)
+	$(MAKE) fullaot-libs AOT_FLAGS="full$(aot_suffix)" GSHAREDVT=$(GSHAREDVT)
```

Comment:
You could also do `full,$(MONO_FULLAOT_ADDITIONAL_ARGS)` here instead then you don't need the Makefile comma escaping hack from above.